### PR TITLE
(feat): Long press button

### PR DESCRIPTION
**Pull request by marjan-georgiev**
- implements ngx-long-press button component
- adds mouse icon

image

### CODE REVIEW

File: src/components/long-press/long-press.component.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+  lastTimeout: any;
+  pressed: boolean = false;
+  pressTimeout: any;
+
+  ngOnInit(): void {
+    this.updateState();
+  }
+
+  ngOnChanges(): void {
```
Comment (amcdnl) [inline src/components/long-press/long-press.component.ts]: Ugh :(

File: src/components/long-press/long-press.component.ts [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+} from '@angular/core';
+import {
+  trigger,
+  state,
+  style,
+  animate,
+  transition
+} from '@angular/animations';
+
+import { ControlValueAccessor, NG_VALUE_ACCESSOR, NgModel } from '@angular/forms';
```
Comment (amcdnl) [inline src/components/long-press/long-press.component.ts]: Unused imports.

File: src/components/long-press/long-press.component.ts [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        strokeDasharray: '1000 1000'
+      })),
+      state('inactive', style({
+        strokeDasharray: '0 1000'
+      })),
+      transition('inactive => active', animate(`{{ duration }}ms ease-out`), {params: {duration: 1000}})
+    ])
+  ]
+})
+export class LongPressComponent implements OnInit, OnChanges {
```
Comment (Hypercubed) [inline src/components/long-press/long-press.component.ts]: Can we break this out into a LongPressDirective (implements `(longPress)`) and the LongPressButtonComponent component?

File: src/components/index.ts [hunk 4]
```diff
@@ -22,3 +22,4 @@ export * from './tree';
 export * from './split';
 export * from './hotkeys';
 export * from './nag';
+export * from './long-press';
```
Comment (Hypercubed) [inline src/components/index.ts]: file should end with a newline

**Pull request merged**